x-pack/plugins/synthetics/public/apps/synthetics/components/getting_started/use_simple_monitor.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useFetcher } from '@kbn/observability-plugin/public';
+import { useEffect } from 'react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { createMonitorAPI } from '../../state/monitor_management/api';
+import { DEFAULT_FIELDS } from '../../../../../common/constants/monitor_defaults';
+import { ConfigKey } from '../../../../../common/constants/monitor_management';
+import { DataStream, SyntheticsMonitorWithId } from '../../../../../common/runtime_types';
+import { MONITOR_SUCCESS_LABEL, MY_FIRST_MONITOR, SimpleFormData } from './simple_monitor_form';
+import { kibanaService } from '../../../../utils/kibana_service';
+
+export const useSimpleMonitor = ({ monitorData }: { monitorData?: SimpleFormData }) => {
+  const { application } = useKibana().services;
+
+  const { data, loading } = useFetcher(() => {
+    if (!monitorData) {
+      return new Promise<undefined>((resolve) => resolve(undefined));
+    }
+    const { urls, locations } = monitorData;
+
+    return createMonitorAPI({
+      monitor: {
+        ...monitorData,
+        ...DEFAULT_FIELDS.browser,
```

Comment:
Seems like defaults should come before monitorData so defaults don't overwrite monitorData.